smart_contracts.rst

```diff
@@ -756,8 +756,10 @@ Finds the endpoint if given a registered Ethereum address.
 - ``eth_address``: The Raiden node's 20 byte Ethereum address.
 
 
-Protocol Overview
-=================
+TokenNetwork Channel Protocol Overview
+======================================
+
+This section contains a few flowcharts on a token network channel.
```

Comment:
Consider: `This section contains a few flowcharts describing the token network channel lifecycle.`

Reply:
That sounds better.  I'll use your sentence in this PR.

Reply:
Done!